cardano-node/src/Cardano/Wallet/Client.hs

```diff
@@ -37,7 +37,7 @@ import           Network.TypedProtocol.Codec
 import           Network.TypedProtocol.Codec.Cbor
 import           Network.TypedProtocol.Driver
 import           Ouroboros.Network.Mux
-import           Ouroboros.Network.Block (Point)
+import           Ouroboros.Network.Block (Tip)
```

Comment:
This means it's not build on `CI`, it would be good to add it so that who ever breaks it is reminded to update it.

Reply:
@coot is there an example script that I can follow to add this to CI?

Reply:
I don't think so, but building it would be enough.